state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -898,6 +903,22 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Update the fee multiplier with the provided value
+		///
+		/// Requires Governance.
+		#[pallet::call_index(7)]
+		#[pallet::weight(T::WeightInfo::ccm_broadcast_failed())]
+		pub fn update_fee_multiplier(
+			origin: OriginFor<T>,
+			new_fee_multiplier: FixedU128,
+		) -> DispatchResult {
+			T::EnsureGovernance::ensure_origin(origin)?;
+
+			FeeMultiplier::<T, I>::put(new_fee_multiplier);
+
```

Comment:
We should emit an event (i think we should add an event `PalletConfigUpdated(PalletConfigUpdate)`)